README.md

```diff
@@ -1,7 +1,82 @@
 # Open MCT
 
-Open MCT is a web-based platform for mission operations user interface
-software.
+Open MCT is a next-generation mission control framework being developed at the NASA Ames Research Center in Silicon Valley. Web-based, for desktop and mobile.
+
+Please visit our [Official Site](https://nasa.github.io/openmct/)
+
+##How is NASA Using Open MCT?
+
+Software based on Open MCT is being used for mission planning and operations in the lead up to the [Resource Prospector mission](https://www.nasa.gov/resource-prospector) and at NASA's Jet Propulsion Laboratory to view data from the Curiosity Rover.
+
+##Features
```

Comment:
Likewise the features section. Again, I really like the idea and a clear list of features is something that the website is lacking, but I'd preferto leave it out of the readme for now.